examples/shardingsphere-sample/shardingsphere-example-generator/src/main/resources/template/java/config/readwrite-splitting.ftl

```diff
@@ -17,8 +17,8 @@
     
     private ReadwriteSplittingRuleConfiguration createReadwriteSplittingRuleConfiguration() {
         Properties props = new Properties();
-        props.setProperties("write-data-source-name", "ds_0");
-        props.setProperties("read-data-source-names", "ds_1, ds_2");
+        props.setProperty("write-data-source-name", "ds_0");
+        props.setProperty("read-data-source-names", "ds_1, ds_2");
```

Comment:
Can we use meaningful name of data source?
`ds_0, ds_1 and ds_2` is not suitable for this case.
I think `write_ds`, `read_ds_0` and `read_ds_1` is better.

Reply:
But for mixed rules, the data source name may not be well defined, and for individual features, we can do additional processing.

Reply:
ok